source/Engine.cpp

```diff
@@ -1196,7 +1208,6 @@ void Engine::EnterSystem()
 	const Date &today = player.GetDate();
 
 	const System *system = flagship->GetSystem();
-	Audio::PlayMusic(system->MusicName());
```

Comment:
Should it not update the music?

Reply:
No, music is now handled by playlists

Reply:
It is just more convenient the other way cause music has to be checked every frame

Reply:
Does it though? Because that would mean rash transitions?
Or mb there is a transition time already coded idk

Reply:
Normally the music is already made to fade out in the end

Reply:
fade out is actually already coded in